### ISSUE TITLE
Why do View.triggers not bubble up through views like DOM events?

### ISSUE THREAD

**Issue by MeoMix** [marionettejs/backbone.marionette#2341]
Hey,

I'm just wondering if there's a specific reason for triggers not bubbling up. They do for CollectionView -> Children, but not for LayoutView -> Region -> ItemView.

Is this just a huge performance concern? Are there other implications I haven't considered? I think that the best way to approach it currently is to use a channel and pass an ID through the channel to ensure that if there's multiple views subscribed to the event that they know which one it is intended for.

For example:

    Grandparenta
        Parent1a
            Child1a
        Parent2a

    Grandparentb
        Parent1b
            Child1b
        Parent2b

Child1a has an event on it and that information needs to get propagated to Parent2a, but not propagate to Parent2b. This requires either a chain of event listeners, or a channel with unique IDs.

Thoughts?

**Comment by jmeas** [comment]
@jasonLaster talked about this the other day. He's working on bubbling events 1 level up from LayoutViews, too, over in #2316.

This will make all events in Marionette bubble up 1 level to their parent. I understand this is a partial solution to the more general problem of event bubbling, but I admit I don't know what's best!

I'd love it if anyone else had thoughts to chime in with. And, as always, thanks for the good issues, @meomix.

**Comment by Tvrqvoise** [comment]
I am against this as a default -- if I listen for a ```show``` event on a view, I likely just want to get the show on that view.  However, I wouldn't mind if we could declare objects to bubble namespaced events from, bindEntityEvents style.

**Comment by Tvrqvoise** [comment]
Adapted from a lil' method I use in a few of my components:
```js
this.listenTo(object, 'all', function(){
  var trigger = this.triggerMethod ? this.triggerMethod : this.trigger;
  var args = Array.prototype.slice.call(arguments);
  args[0] = namespace + ':' + args[0];
  trigger.apply(this, args);
});
```

**Comment by ahumphreys87** [comment]
Fixed with #2316
